ui/.eslintrc.js

```diff
@@ -39,10 +39,6 @@ module.exports = {
     "keyword-spacing" : "error",
     "no-trailing-spaces" : "error",
     "object-curly-spacing" : ["error", "always"],
-    "arrow-spacing" : ["error", {
-      "before" : true,
-      "after" : true,
-    }],
```

Comment:
don't need to make changes here. Please revert.

Reply:
oh ok done, I notice their order just now and feel how different they are :+1: